coch.py

```diff
@@ -0,0 +1,53 @@
+import os
+import sys
+import argparse
+
+def main(mode='rand_sound'):
+    mode = mode.lower()
+
+    from pycochleagram import cochleagram as cgram
+    from pycochleagram import erbfilter as erb
+    from pycochleagram import utils
+    from random import choice
+    from pycochleagram.demo import demo_human_cochleagram_helper
+
+    from os.path import dirname, join, realpath
+    from pydub import AudioSegment
+    # from argparse import argumentparser
+  #AUDIO PROCESSING SECTION: MANIPULATES MP3 FILES AND CONVERTS TO WAV, SLICES $
+    DEMO_PATH = join(dirname(realpath(__file__)) #'/media/sf_acm_mirum_tempo' THE DATASET LINK MUST BE PUT HERE
```

Comment:
It may be helpful to put a small dataset in version control and include the relative path here, so that the code will run without modification.